docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -85,7 +108,7 @@ But `async` and `await` are the real stars of this release. When these features
 
 ## C# version 6.0
 
-With versions 3.0 and 5.0, C# had added some impressive features in an object oriented language. With version 6.0, it would go away from doing a dominant killer feature and instead release many features that delighted users of the language. Here are some of them:
+With versions 3.0 and 5.0, C# had added some impressive features in an object-oriented language. With version 6.0, it would go away from doing a dominant killer feature and instead release many features that delighted users of the language. Here are some of them:
```

Comment:
impressive and delighted seem to me like market speak. Perhaps simply major and incremental or something similar?